src/azure-cli/azure/cli/command_modules/storage/_help.py

```diff
@@ -814,6 +814,11 @@
 short-summary: Manage blob metadata.
 """
 
+helps['storage blob query'] = """
+type: command
+short-summary: Enable users to select/project on blob/or blob snapshot data by providing simple query expressions.
```

Comment:
`select/project on blob/or blob snapshot` This means "select project on blob or blob snapshot", right? What does `/` mean?

Reply:
updated